src/components/Comment/Comment.tsx

```diff
@@ -507,24 +508,28 @@ export const Comment = ({
                           onClick={() => setCommentBeingRepliedTo(comment)}
                           className={cx(
                             commentControlsButtonStyles,
-                            removePaddingLeft
+                            removePaddingLeft,
+                            flexRowStyles
                           )}
                         >
-                          <div className={flexRowStyles}>
-                            <ReplyArrow />
-                            Reply
-                          </div>
+                          <ReplyArrow />
+                          Reply
                         </button>
                       ) : (
-                        <Row>
+                        <div
+                          className={cx(
+                            commentControlsButtonStyles,
+                            flexRowStyles
+                          )}
+                        >
```

Comment:
This could be a Button. Then we can pass the svg in as a prop. We actually already have this reply button in the code here

https://github.com/guardian/discussion-rendering/blob/577c602da6a650a4605e53d6772e450e35229fdb/src/components/Comment/Comment.tsx#L370